### PR TITLE
Fix images accessibility violations

### PR DESCRIPTION
**Pull request by abdulbasithqb**
#### What does this PR do?
` fixes: images accessibility violations`
#### Where should the reviewer start?
storybook
#### What testing has been done on this PR?
yarn test
#### How should this be manually tested?
Check the accessibility tab in the images/all-variants storybook
#### Do Jest tests follow these best practices?

- [ ] `screen` is used for querying.
- [ ] The correct query is used. (Refer to [this list of queries](https://testing-library.com/docs/queries/about/#priority))
- [ ] `asFragment()` is used for snapshot testing.

#### What are the relevant issues?
 #6124
#### Screenshots (if appropriate)

Screenshot 2023-12-13 at 5 10 00 PM

#### Do the grommet docs need to be updated?
No
#### Should this PR be mentioned in the release notes?
No
#### Is this change backwards compatible or is it a breaking change?
No

### CODE REVIEW

**Review by britt6612 — APPROVED** [review]
LGTM!
Thanks for your contribution.

File: src/js/components/Image/stories/Simple.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { Image } from 'grommet';
 export const Simple = () => (
   // Uncomment <Grommet> lines when using outside of storybook
   // <Grommet theme={...}>
-  <Image src="//v2.grommet.io/assets/IMG_4245.jpg" />
+  <Image src="//v2.grommet.io/assets/IMG_4245.jpg" alt="image" />
```
Comment (jcfilben) [inline src/js/components/Image/stories/Simple.js:8]: Can this be changed so that the alt text has a more meaningful description for screenreader users?

Comment (abdulbasithqb) [reply src/js/components/Image/stories/Simple.js]: update the alt text

**Review by jcfilben — APPROVED** [review]

**Comment by jcfilben** [comment]
Hey @abdulbasithqb I can't update your branch with the master branch. Can you either give maintainers access or update this PR with the latest changes from the master branch of Grommet?

**Comment by abdulbasithqb** [comment]
> Hey @abdulbasithqb I can't update your branch with the master branch. Can you either give maintainers access or update this PR with the latest changes from the master branch of Grommet?

Hey @jcfilben, I have updated the branch with master

**Pull request merged**